tests/foreman/api/test_product.py

```diff
@@ -122,3 +123,42 @@ def test_positive_update_1(self, attrs):
         for name, value in attrs.items():
             self.assertIn(name, new_attrs.keys())
             self.assertEqual(value, new_attrs[name])
+
+
+@run_only_on('sat')
+class RepositorySetsTestCase(APITestCase):
+    """Tests for ``katello/api/v2/products/<product_id>/repository_sets``."""
+
+    def test_repositoryset_enable(self):
+        """@Test: Enable repo from reposet by reposet, org and product
+
+        @Feature: Repository-set
+
+        @Assert: Repository was enabled
+
+        """
+        org = entities.Organization().create()
+        manifest_path = manifests.clone()
+        org.upload_manifest(path=manifest_path)
+        prd_id = entities.Product().fetch_rhproduct_id(
+            name=PRDS['rhel'],
+            org_id=org.id,
+        )
+        reposet_id = entities.Product(id=prd_id).fetch_reposet_id(
+            name=REPOSET['rhva6'],
+        )
+        entities.Product(id=prd_id).enable_rhrepo(
+            base_arch='x86_64',
+            release_ver='6Server',
+            reposet_id=reposet_id,
+        )
+        result = entities.Product(id=prd_id).list_available_rhrepos(
```

Comment:
As per https://github.com/SatelliteQE/nailgun/pull/122#discussion_r32751154, can you update this method call?